oojs.js

```diff
@@ -3,10 +3,26 @@
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
-
+class Playlist{
+  constructor(title){
+    this.title = title;
+    this.favorites = 0;
+    this.songs = [];
```

Comment:
`songs` should be "determined by some input", meaning that it should be passed as an argument to the class/constructor. So in the end you would say `this.songs = songs`.